.gitignore

```diff
@@ -3,6 +3,13 @@
 ##
 ## Get latest from https://github.com/github/gitignore/blob/master/VisualStudio.gitignore
 
+# REHITMAN SPECIFIC FOLDER IGNORES
```

Comment:
Do you really want to ignore that? 
It makes impossible to upload any changes.

Reply:
Hrmm - Good point now that I think about it.
Thanks for the review